.github/workflows/release.yaml

```diff
@@ -37,6 +37,8 @@ jobs:
       - uses: actions/setup-go@v5
         with:
           go-version-file: go.mod
+      - run: |
```

Comment:
It's a bit tedious decorating every `go build` command with this, but it seemed better than checking all the files into the repo. Let me know if you have any thoughts